README.md

```diff
@@ -13,4 +13,120 @@ layout component for the home page and all slug pages, but not in the preview ro
 ### Vercel
 Production environment uses the `main` branch. `1.x` branch is used as a release branch for all features. Any commits
 made to the `1.x` branch will trigger GitHub actions to update the `dev` and `test` branches which are then deployed
-on Vercel for their respective Drupal environments.
\ No newline at end of file
+on Vercel for their respective Drupal environments.
+
+## Pull Request and Development Workflow Guide
+
+### Development Workflows
+
+All general development work should be based off of the `1.x` branch. To maintain organization and consistency, branches must follow this naming convention:
+
+```markdown
+
+{branch-type}/JIRA-###--optional-description-of-task
+
+```
+
+- **branch-type**: Use `feature/`, `task/`, or `bug/` to describe the branch purpose.
+
+- **JIRA-###**: Corresponds to the Jira ticket number.
+
+- **optional-description-of-task**: A brief description of the task or feature being implemented.
+
+This structure ensures proper labeling and integration with Jira.
```

Comment:
At least one commit needs to contain the jira ticket. I don't believe the branch name will actually do any integration.